taskchampion/src/taskdb/undo.rs

```diff
@@ -90,16 +90,16 @@ pub fn commit_reversed_operations(txn: &mut dyn StorageTxn, undo_ops: Operations
 
```

Comment:
The `remove_operation` might fail for other reasons than inconsistent operations, so that could lead to incorrect error messages. I think it's best to still compare the lists before beginning. But, you're right that it can be simplified.